### PR TITLE
works with any image size; code cleanup (logging etc.)

### PR DESCRIPTION
**Pull request by AndreaCensi**

### CODE REVIEW

**Comment by AndreaCensi** [comment]
Tests fail because also slimremote needs to be updated.

**Comment by AndreaCensi** [comment]
@fgolemo can I merge?

**Comment by AndreaCensi** [comment]
@liampaull can we merge this?

File: gym_duckietown_agent/envs/simplesimagent_env.py [hunk 1]
```diff
@@ -22,7 +20,7 @@ class SimpleSimAgentEnv(gym.Env):
         'video.frames_per_second': 30  # TODO: do we need this on the client?
     }
 
-    def __init__(self, silent=False):
+    def __init__(self, silent, camera_width, camera_height):
```
Comment (liampaull) [inline gym_duckietown_agent/envs/simplesimagent_env.py:23]: @AndreaCensi why take away False from silent=False - will it fail if called without a 2nd parameter now ?

Comment (AndreaCensi) [reply gym_duckietown_agent/envs/simplesimagent_env.py:23]: I should have changed the calls. But don't have strong feelings about it.

(For me default arguments should carry the assurance that the defaults never change. This in practice is not respected so better make everything explicit.)

**Pull request merged**